src/components/NewMessage/NewMessagePollEditor.vue

```diff
@@ -194,6 +194,7 @@ export default {
 		align-items: center;
 		width: 100%;
 		height: 44px;
+		margin-top: 4px;
```

Comment:
Maybe 12?
It's overlapping now
![image](https://github.com/nextcloud/spreed/assets/93392545/50ee19a4-e2fb-4ee1-b6af-80f98e165649)